diff -u --recursive --new-file SNNSv4.1/kernel/sources/Makefile SNNSv4.1.me/kernel/sources/Makefile
--- SNNSv4.1/kernel/sources/Makefile	Fri Dec  1 00:00:00 1995
+++ SNNSv4.1.me/kernel/sources/Makefile	Mon Aug  4 20:28:53 1997
@@ -21,17 +21,17 @@
 # Created by mamier at Tue Nov 21 15:10:13 MET 1995
 
 # -- SNNS  Installation  Directories  &  Libraries --
-SNNSDIR = /usr/local/bv/SNNS/SNNSv4.1
-MACHINE = sun4
+SNNSDIR = /usr/lib/SNNS
+MACHINE = 386
 LEX = flex
-MACHINENAME= sun_solaris
+MACHINENAME= pc_linux
 COMPILER = gcc
 CCFLAGS = -O4
 FPOPTIONS = 
-LEXLIB = -ll
-RLIB = echo
-XDIR = /usr/local/X11R6/lib
-XH = /usr/local/X11R6/include
+LEXLIB = -lfl
+RLIB = ranlib
+XDIR = /usr/X11R6/lib
+XH = /usr/X11R6/include
 
 #*****************************************************************************
 # FILE           : $Source$
@@ -73,7 +73,7 @@
 
 # -- Archive Configuration --
 
-CP		= cp
+CP		= cp -a
 MV              = mv
 PATHDEL		= /
 AR              = ar
@@ -219,8 +219,9 @@
 	-$(RANLIB) $(LIBFUNCS)
 
 install: $(LIBKERNEL) $(LIBFUNCS)
-	$(MV) $(LIBKERNEL) $(KERNELLIBDIR)
-	$(MV) $(LIBFUNCS) $(KERNELLIBDIR)
+	mkdir -p $(DESTDIR)/$(KERNELLIBDIR)
+	$(CP) $(LIBKERNEL) $(DESTDIR)/$(KERNELLIBDIR)
+	$(CP) $(LIBFUNCS) $(DESTDIR)/$(KERNELLIBDIR)
 
 parser: 
 	@echo "You need to use bison version 1.22 or later. Your version is:"
diff -u --recursive --new-file SNNSv4.1/tools/sources/Makefile SNNSv4.1.me/tools/sources/Makefile
--- SNNSv4.1/tools/sources/Makefile	Fri Dec  1 00:00:00 1995
+++ SNNSv4.1.me/tools/sources/Makefile	Mon Aug  4 20:29:54 1997
@@ -21,18 +21,18 @@
 # Created by mamier at Tue Nov 21 15:10:13 MET 1995
 
 # -- SNNS  Installation  Directories  &  Libraries --
-SNNSDIR = /usr/local/bv/SNNS/SNNSv4.1
-MACHINE = sun4
+SNNSDIR = /usr/lib/SNNS
+MACHINE = 386
 LEX = flex
-MACHINENAME= sun_solaris
+MACHINENAME= pc_linux
 COMPILER = gcc
 CCFLAGS = -O4
 FPOPTIONS = 
-LEXLIB = -ll
-RLIB = echo
-XDIR = /usr/local/X11R6/lib
-XH = /usr/local/X11R6/include
-LDOPTIONS = -R$(XDIR) -ldl -lc -lw -lnsl -L$(XDIR) -lXaw -lXmu -lXt -lSM -lICE -lXext -lX11 -lm -lsocket -ll
+LEXLIB = -lfl
+RLIB = ranlib
+XDIR = /usr/X11R6/lib
+XH = /usr/X11R6/include
+LDOPTIONS = -lc -L$(XDIR) -lXaw -lXmu -lXt -lSM -lICE -lXext -lX11 -lm -lfl
 
 #*****************************************************************************
 # FILE           : $Source$
@@ -55,8 +55,8 @@
 #*****************************************************************************
 
 TOOLSBINDIR	= $(SNNSDIR)/tools/bin/$(MACHINENAME)
-KERNELLIBDIR	= $(SNNSDIR)/kernel/bin/$(MACHINENAME)
-KERNELSOURCES	= $(SNNSDIR)/kernel/sources
+KERNELLIBDIR	= ../../kernel/sources
+KERNELSOURCES	= ../../kernel/sources
 
 # -- C-Compiler Configuration --
 
@@ -67,7 +67,7 @@
 
 # -- Archive Configuration --
 
-CP		= cp
+CP		= cp -a
 PATHDEL		= /
 AR              = ar
 ARFLAGS 	= rc
@@ -198,7 +198,7 @@
 	$(CC) $(OPTIONS) $(CFLAGS) -c icopjmp.c
 
 ic_snns.o: ic_snns.c ic_snns.ph symtab.h batchman.h ictab.h error.h 
-	$(CC) $(OPTIONS) $(CFLAGS) -I$(SNNSDIR)/xgui/sources \
+	$(CC) $(OPTIONS) $(CFLAGS) -I../../xgui/sources \
 	-I$(XH) -c ic_snns.c
 
 arglist.o: arglist.c arglist.ph symtab.h ictab.h error.h 
@@ -238,9 +238,10 @@
 
 install: snns2c netlearn bignet netperf snnsbat feedback-gennet convert2snns \
 	isnns analyze pat_sel mkhead mkout mkpat batchman
+	mkdir -p $(DESTDIR)/$(TOOLSBINDIR)
 	$(CP) snns2c netlearn bignet netperf snnsbat feedback-gennet \
 	 convert2snns isnns analyze pat_sel mkhead mkout mkpat\
-	 batchman $(TOOLSBINDIR) 
+	 batchman $(DESTDIR)/$(TOOLSBINDIR) 
 
 list:
 	cat $(SOURCES) > toolssrc.all
diff -u --recursive --new-file SNNSv4.1/xgui/sources/Makefile SNNSv4.1.me/xgui/sources/Makefile
--- SNNSv4.1/xgui/sources/Makefile	Fri Dec  1 00:00:00 1995
+++ SNNSv4.1.me/xgui/sources/Makefile	Mon Aug  4 20:30:22 1997
@@ -21,18 +21,18 @@
 # Created by mamier at Tue Nov 21 15:10:13 MET 1995
 
 # -- SNNS  Installation  Directories  &  Libraries --
-SNNSDIR = /usr/local/bv/SNNS/SNNSv4.1
-MACHINE = sun4
+SNNSDIR = /usr/lib/SNNS
+MACHINE = 386
 LEX = flex
-MACHINENAME= sun_solaris
+MACHINENAME= pc_linux
 COMPILER = gcc
 CCFLAGS = -O4
 FPOPTIONS = 
-LEXLIB = -ll
-RLIB = echo
-XDIR = /usr/local/X11R6/lib
-XH = /usr/local/X11R6/include
-LDOPTIONS = -R$(XDIR) -ldl -lc -lw -lnsl -L$(XDIR) -lXaw -lXmu -lXt -lSM -lICE -lXext -lX11 -lm -lsocket -ll
+LEXLIB = -lfl
+RLIB = ranlib
+XDIR = /usr/X11R6/lib
+XH = /usr/X11R6/include
+LDOPTIONS = -lc -L$(XDIR) -lXaw -lXmu -lXt -lSM -lICE -lXext -lX11 -lm -lfl
 
 #*****************************************************************************
 # FILE           : $Source$
@@ -57,22 +57,22 @@
 XGUIBINDIR= $(SNNSDIR)/xgui/bin/$(MACHINENAME)
 XGUILIBDIR= $(SNNSDIR)/xgui/bin/$(MACHINENAME)
 PROGRAM   = xgui
-LIBKERNEL = $(SNNSDIR)/kernel/bin/$(MACHINENAME)/libkernel.a \
-	$(SNNSDIR)/kernel/bin/$(MACHINENAME)/libfunc.a
+LIBKERNEL = ../../kernel/sources/libkernel.a \
+	../../kernel/sources/libfunc.a
 
 
 # -- C-Compiler Configuration --
 
 CC              = $(COMPILER)
-CPPFLAGS        = -I$(SNNSDIR)/kernel/sources \
-                  -I$(SNNSDIR)/xgui/iconsXgui \
+CPPFLAGS        = -I../../kernel/sources \
+                  -I../../xgui/iconsXgui \
                   -I$(XH)
 CFLAGS          = $(CDEBUG) $(CCFLAGS) $(FPOPTIONS) $(CPPFLAGS)
 CDEPFLG         = -MM
 
 # -- Archive Configuration --
 
-CP		= cp
+CP		= cp -a
 MV		= mv
 PATHDEL		= /
 AR              = ar
@@ -293,11 +293,12 @@
 	-$(RANLIB) $(BN3DLIB)
 
 install: $(PROGRAM)
-	$(CP) $(PROGRAM) $(XGUIBINDIR)
-	$(MV) $(LIBXGUI1) $(XGUIBINDIR)
-	$(MV) $(LIBXGUI2) $(XGUIBINDIR)
-	$(MV) $(BN3DLIB) $(XGUIBINDIR)
-	-strip $(XGUIBINDIR)/$(PROGRAM)
+	mkdir -p $(DESTDIR)/$(XGUIBINDIR)
+	$(CP) $(PROGRAM) $(DESTDIR)/$(XGUIBINDIR)
+	$(CP) $(LIBXGUI1) $(DESTDIR)/$(XGUIBINDIR)
+	$(CP) $(LIBXGUI2) $(DESTDIR)/$(XGUIBINDIR)
+	$(CP) $(BN3DLIB) $(DESTDIR)/$(XGUIBINDIR)
+	-strip $(DESTDIR)/$(XGUIBINDIR)/$(PROGRAM)
 
 list:
 	cat $(SOURCES) >  xguisrc.all
